src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import React from 'react';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+
+    <hr />
+
+    {post.comments.length > 0 ? (
+      <CommentList comments={post.comments} />
```

Comment:
According to the checklist, you need to add a `key` to the root tag inside an iteration. If `CommentList` is iterating over comments internally, ensure that each comment item has a unique `key` prop.